### ISSUE TITLE
Allow the declaration of multiple licenses

### ISSUE THREAD

**Issue by danielleberre** [codemeta/codemeta-generator#12]
My software is licensed under the dual GNU LGPL 2.1 and EPL 1.0 licenses.

It is currently not possible to declare multiple licenses in codemeta-generator.

I had to select one of them, which does not reflect the reality: https://gitlab.ow2.org/sat4j/sat4j/-/commit/b939a8168ee67047e0236464153bdad6bfc6e8f5

**Comment by progval** [comment]
This will require some UI redesign to do this nicely (same issue as #3)

In the meantime, you can write this manually:

```
"license": [
    "https://spdx.org/licenses/EPL-1.0",
    "https://spdx.org/licenses/LGPL-2.1"
],
```

**Comment by progval** [comment]
yes, but the codemeta-generator currently does not support importing them

**Comment by progval** [comment]
Resolved by #15
